application/templates/_helpers.tpl

```diff
@@ -39,10 +39,8 @@ Common labels
 */}}
 {{- define "application.labels" -}}
 helm.sh/chart: {{ include "application.chart" . }}
-app.kubernetes.io/name: {{ include "application.name" . }}
 app.kubernetes.io/version: {{ include "application.version" . }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
-app.kubernetes.io/component: {{ include "application.name" . }}
```

Comment:
@mustafaStakater if someone isn't using argocd then what will happen?

Reply:
it shouldnt be a problem in that case, i left part-of label for users to identify what application is this a part of